xml/_filter.xml

```diff
@@ -153,9 +156,11 @@
       <typeFilter name="*" expose="false" />
     </namespaceFilter>
     <namespaceFilter name="Microsoft.Extensions.Logging.Console.Internal">
+      <typeFilter name="IConsole" expose="true" />
```

Comment:
This looks to be `internal`. And I don't see it in this namespace. I think it should be reverted.

Reply:
Ahh okay. Unfortunately it's the property type for [ConsoleLogger.Console](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.console.consolelogger.console?view=dotnet-plat-ext-2.2#Microsoft_Extensions_Logging_Console_ConsoleLogger_Console).

Reply:
What version are you looking at? It looks like 2.2, which is pretty old.

In the latest (5.0) version of `microsoft.extensions.logging.console`, the `ConsoleLogger` class is not public.

Reply:
Yes, 2.2. I have reverted all except ISystemClock (just haven't committed yet).